$NetBSD: patch-configure.in,v 1.1 2013/06/29 20:29:47 tsutsui Exp $

- pull post-3.2.1 fixes:
  http://mlterm.sf.net/mlterm-3.2.1-fixes.patch
  > Fix configure error which --with-gui=xlib,fb option causes.
  > (enbugged at 3.2.1)

--- configure.in.orig	2013-06-29 12:45:20.000000000 +0000
+++ configure.in
@@ -82,11 +82,16 @@ if test "$gui" = "fb,xlib" -o "$gui" = "
 	fi
 	mkdir -p ${top_srcdir-$srcdir}/fb
 	for file in ChangeLog Makefile.in common configure etc gtk inputmethod java \
-		main man mlterm script scrollbar xwindow ; do
+		main man mlterm script scrollbar xwindow doc ; do
 		cp -R ${top_srcdir-$srcdir}/$file ${top_srcdir-$srcdir}/fb/
 	done
 fi
 
+if test "$gui" = "fb" ; then
+	MAKE_DIRS="doc/kbd ${MAKE_DIRS}"
+	OUTPUT_FILES="doc/kbd/Makefile ${OUTPUT_FILES}"
+fi
+
 ml_cv_is_posix="yes"
 
 case "${host_os}" in
@@ -1094,8 +1099,8 @@ if test "${SUBDIRS}" != "" ; then
 	AC_CONFIG_SUBDIRS(${SUBDIRS})
 fi
 
-MAKE_DIRS="mlterm mlterm/libctl xwindow/libtype xwindow main man etc doc/kbd ${MAKE_DIRS}"
-OUTPUT_FILES="Makefile mlterm/Makefile mlterm/libctl/Makefile xwindow/libtype/Makefile xwindow/Makefile main/Makefile man/Makefile etc/Makefile java/Makefile doc/kbd/Makefile ${OUTPUT_FILES}"
+MAKE_DIRS="mlterm mlterm/libctl xwindow/libtype xwindow main man etc ${MAKE_DIRS}"
+OUTPUT_FILES="Makefile mlterm/Makefile mlterm/libctl/Makefile xwindow/libtype/Makefile xwindow/Makefile main/Makefile man/Makefile etc/Makefile java/Makefile ${OUTPUT_FILES}"
 
 AC_ARG_ENABLE(ind,
 	[  --enable-ind            libind @<:@default=disabled@:>@],
